### PR TITLE
Expose unknown tags

### PR DESCRIPTION
**Pull request by rafaelcaricio**
When I was using the library I noticed the unknown tags are being parsed but are not exposed. Maybe this is something other people would be interested as well?

### CODE REVIEW

File: src/playlist.rs [hunk 1]
```diff
@@ -584,6 +592,8 @@ pub struct MediaSegment {
     pub program_date_time: Option<String>,
     /// `#EXT-X-DATERANGE:<attribute-list>`
     pub daterange: Option<String>,
+    /// `#EXT-` that are unknown or customized
+    pub unknown: Option<Vec<ExtTag>>,
```
Comment (rutgersc) [inline src/playlist.rs:596]: It's fine to just use a `Vec<ExtTag>` instead of `Option<Vec<ExtTag>>` here

**Comment by rutgersc** [comment]
Sounds like a good addition to me. The build is still fails on one test though, I believe that's because the unkown tags aren't written out yet, see https://github.com/rutgersc/m3u8-rs/blob/43f68b643b337d14db275afa0fedb12c67d996ac/src/playlist.rs#L605.

File: src/playlist.rs (hunk 1 above)
Comment (rutgersc) [reply src/playlist.rs:596]: Actually, after looking at #19, I think it's better to move the unkown tags up to `MasterPlaylist` and `MediaPlaylist`. Let me know if you want some help or if I can push the changes to your branch

Comment (rafaelcaricio) [reply src/playlist.rs:596]: thank you for the feedback @rutgersc ! If you want to make the change, please feel free. I can have a look later if this is still open.

Comment (rutgersc) [reply src/playlist.rs]: Alright, i've pushed the changes 👍

**Comment by rafaelcaricio** [comment]
Thank you @rutgersc ! 🥳

**Pull request merged**